tslint.json

```diff
@@ -1,99 +1,76 @@
 {
-    "extends": ["tslint-react"],
```

Comment:
Were there actual changes or are these all prettier diffs?

Reply:
there were actual changes. I can separate this into a another pr though.

Reply:
Nah, doesn't matter at this stage, just curious what they were
Feel free to merge

Reply:
👍 line length and semi were two of them.